build.gradle

```diff
@@ -44,3 +44,7 @@ checkstyle {
 run{
     standardInput = System.in
 }
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
```

Comment:
Quite interesting that you added this to gradle properties to be able to render Ω, μ. Hadn't thought of this before!